src/main/java/com/limechain/rpc/methods/childstate/ChildStateRPCImpl.java

```diff
@@ -15,8 +16,9 @@
 
 @Service
 public class ChildStateRPCImpl {
+    @Autowired
```

Comment:
I agree completely. In general, the project needs some sort of convention on how we'll do dependencies, because now it's a mixture of spring beans (mainly used outside of spring components), static singletons, and, well..., the approach introduced in this PR (explicit manual context definition and management). 

This changes was introduced as a minimal quick fix in order to just push the boundaries as little as necessary for the purposes of sandboxing the runtime invocations.

Honestly, it was one of those small needed changes that would snowball into an avalanche of refactoring, so I spared this effort for this PR, since it'd quickly pivot into the above mentioned separate topic (how we do dependencies in general).